.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ""
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+<!-- A clear and concise description of what the bug is. -->
+
+**To Reproduce**
+<!-- Steps to reproduce the behavior: -->
+1. [e.g. Go to '...']
+2. [e.g. Click on '....']
+3. [e.g. Scroll down to '....']
+4. [e.g. See error]
+
+**Expected behavior**
+<!-- A clear and concise description of what you expected to happen. -->
+
+**Screenshots**
+<!-- If applicable, add screenshots to help explain your problem. -->
+
+**Host**
+<!-- Please complete the following information -->
+ - OS: [e.g. Windows, Ubuntu]
+ - Architecture [e.g. 32 bit, 64 bit, arm]
+ - Version [e.g. 0.11.1]
+
```

Comment:
I would add here other important informations for GPU and host. On Windows the driver version and the GPU Vendor (Intel, AMD, Nvidia and the model). On Linux the capture method (PipeWire/KVM/X11) and the appropriate info (like the Mesa Version)

Reply:
How about this?

```yaml
**Host**
<!-- Please complete the following information -->
 - OS: [e.g. Windows, Linux, Mac... include build/distro details]
 - Architecture: [e.g. 32 bit, 64 bit, arm]
 - Version: [e.g. 0.11.1]
 - GPU Type: [e.g. Intel, AMD, Nvidia]
 - GPU Model:
 - GPU Driver/Mesa Version:
 - Capture method (Linux only): [e.g. PipeWire/KVM/X11]
```

Reply:
Yup, I like this!

Reply:
@TheElixZammuto we good to merge this?